sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -262,7 +262,11 @@ export class ServiceBusReceiverImpl<
         try {
           await onInitialize();
         } catch (err) {
-          onError(err);
+          onError(err, {
+            errorSource: "processMessageCallback",
```

Comment:
Not exactly from the message callback, right?

Reply:
Changed over to 'receive', which I think fits better - since this is only internally facing errors that somehow occur in here do fall under the general umbrella of internal failures.